docs/en/siem/field-ref.asciidoc

```diff
@@ -2,15 +2,17 @@
 [chapter, role="xpack"]
 = Field reference and mappings
 
-This section lists all fields required to populate {siem-soln} data.
+This section lists the fields required to populate all {siem-soln} data.
 
-NOTE: Unless noted otherwise, all listed fields are {ecs-ref}[ECS fields]. 
+NOTE: Unless noted otherwise, all listed fields are {ecs-ref}[ECS fields].
+Whenever possible, it is recommended to use ECS fields. In future releases,
+{siem-soln} will adopt more ECS fields and non-ECS fields will be deprecated. To reduce time spent on updating you ingestion pipelines in the future, see {ecs-ref}/ecs-converting.html[how to convert data to ECS].
```

Comment:
- suggest removing "whenever possible" to make this more clear and authoritative.
- suggest changing "will adopt" to "is expected to adopt"
- suggest removing the mention of deprecation, as we're not sure if we'll actually deprecate their use.  It is true that we hope to remove all dependencies on non-ECS fields, but that's not really the same thing.
- suggest change "updating you" to "updating your"

I think there's still a message that is not getting across.  Here's what I think it should say, in rough terms:

> Here's the list of fields required by the current version of the SIEM app. (This should have very few, if any, non-ECS fields)
> When building your ingest pipelines, do NOT limit your implementation to only these required fields, but instead populate as many ECS fields as possible/practical, since future versions of the SIEM app are expected to adopt additional ECS fields, and if you do only the required ones, you'll have to update your ingest pipelines in the future.

Reply:
Thanks @MikePaquette - I've updated the text slightly (and fixed the typo), but I think this complicates the issue for users. After this PR is merged, only required non-ECS fields will be listed. If we start suggesting that users should create a futureproof pipeline, do we need to give them more concrete information? If their pipeline now maps 1 source field to 2 target fields (one ECS and one non-ECS), won't they want to fix that anyway?

Reply:
Updated, @MikePaquette please let me know if the new text is OK with you. thanks